doc/nrf/releases/release-notes-latest.rst

```diff
@@ -34,7 +34,12 @@ Zigbee
 nRF9160
 =======
 
+Modem libraries
+---------------
 
+* :ref:`nrf_modem_lib_readme` library:
+
+  * Fixed a bug in the socket offloading component, where the :c:func:`recvfrom` wrapper could do an out-of-bounds copy of the sender's address, when the application is compiled without IPv6 support. In some cases, the out of bounds copy could indefinitely block the :c:func:`send` and other socket API calls.
```

Comment:
We don't have a process of editing the change log of older releases. As and when a release happens, we clean up the changelog and convert it to a release note. The change log is then orphaned and not made part of the release documentation. Usually, it is not edited further until we have a release from that branch. 
cc: @carlescufi , @umapraseeda , @b-gent

Reply:
I really don't know. @umapraseeda and @b-gent could you please weigh in?

Reply:
@divipillai  you are right, in such case we do write a known issue in main and add previous release tag  with suggested work around.